Lacework/lacework_module/base.py

```diff
@@ -4,9 +4,9 @@
 
 
 class LaceworkConfiguration(BaseModel):
-    secret_key: str = Field(secret=True, description="Your unique identifier used as the Authorization header")
-    access_key: str = Field(..., description="Your unique key to create the token used to authenticate the API Key.")
-    lacework_url: str = Field(..., description="Your Lacework application name")
+    secret: str = Field(secret=True, description="The secret of your API Key")
```

Comment:
```suggestion
    secret: str = Field(..., description="The secret of your API Key", secret=True)
```
The field still has to be required (though, idk whether this change will affect anything)